scale/job/execution/configuration/configurators.py

```diff
@@ -340,7 +342,25 @@ def _configure_all_tasks(self, config, job_exe, job_type):
                         volume = Volume(vol_name, cont_path, task_workspace.mode, is_host=False, driver=driver,
                                         driver_opts=driver_opts)
                     workspace_volumes[task_workspace.name] = volume
-
+            
+            
+            # Configure the input_metadata env variable
+            input_metadata = {}
+            if 'input_files' in config._configuration:
+                input_metadata['JOB'] = {}
+                for i in config._configuration['input_files'].keys():
+                    input_metadata['JOB'][i] = [ScaleFile.objects.get(pk=f['id'])._get_url() for f in config._configuration['input_files'][i]]
```

Comment:
It might not need to be handled like metadata is being handled here to pass urls along but we should be passing in json values from the recipe to jobs somehow.  We should make sure that's being done elsewhere or add it here.

Reply:
Updated this to pull the input data information from the job_exe.job as follows:
```
input_data = job_exe.job.get_input_data()
for i in input_data.values.keys():
      if type(input_data.values[i]) is JsonValue:
         input_metadata['JOB'][i] = input_data.values[i].value
      elif type(input_data.values[i]) is FileValue:
         input_metadata['JOB'][i] = [ScaleFile.objects.get(pk=f)._get_url() for f in input_data.values[i].file_ids]
```

This should take care of passing any json inputs, as well as input file URLs.